### PR TITLE
Bump: Python 3.8 & Ansible-core 2.11.3 & action-molecule-avd 1.3 & update requirement to minimum version

### PR DESCRIPTION
**Pull request by carlbuchmann**
## Change Summary

Update python requirements to set minimum version only.

## Related Issue(s)

Includes PR #1179 

## Component(s) name

Collection python requirements

## Proposed changes

PR #1179 
+ set python requirements from `==` to `>=`
+ bump `action-molecule-avd` to v1.3

## How to test

Create new venv with new requirements
Note, new ansible-core requirement requires to uninstall current ansible version and then install ansible-core 

### User Checklist

- [x] Create new venv and install requirements and requirement-dev
- [x] Run molecule scenarios `make refresh-facts`
- [x] Functional test with avd lab

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Comment by titom73** [comment]
Linting to fix:

```
ERROR: Found 3 no-smart-quotes issue(s) which need to be resolved:
See documentation for help: docs.ansible.com/ansible/2.11/dev_guide/testing/sanity/no-smart-quotes.html
ERROR: roles/eos_cli_config_gen/README.md:1552:101: use ASCII quotes `'` and `"` instead of Unicode quotes
ERROR: roles/eos_cli_config_gen/README.md:1557:80: use ASCII quotes `'` and `"` instead of Unicode quotes
ERROR: roles/eos_cli_config_gen/README.md:1563:80: use ASCII quotes `'` and `"` instead of Unicode quotes
```

File: .github/workflows/continuous_integration.yml [hunk 1]
```diff
@@ -155,12 +155,13 @@ jobs:
           echo "ANSIBLE_FORCE_COLOR=1" >> $GITHUB_ENV
       - uses: actions/checkout@v2
       - name: Run molecule action
-        uses: arista-netdevops-community/action-molecule-avd@v1.2
+        uses: arista-netdevops-community/action-molecule-avd@master
```
Comment (titom73) [inline .github/workflows/continuous_integration.yml]: We should target `v1.3` as version has been released

File: ansible_collections/arista/avd/molecule/dhcp_provisionning/molecule.yml [hunk 2]
```diff
@@ -30,7 +30,7 @@ driver:
   name: docker
 platforms:
   - name: DC1-LEAF1A
-    image: avdteam/base:3.6
+    image: avdteam/base:3.8
```
Comment (titom73) [inline ansible_collections/arista/avd/molecule/dhcp_provisionning/molecule.yml]: We should converge all base image to the same and `3.8-v2.0` is more up-to-date than the old one

File: development/docker-compose.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 version: "3"
 services:
   ansible:
-    image: avdteam/base:3.6
+    image: avdteam/base:3.8
```
Comment (titom73) [inline development/docker-compose.yml]: Same here, better to use `3.8-v2.0`

File: development/Makefile [hunk 4]
```diff
@@ -3,7 +3,7 @@
 SHELL := /bin/bash
 # AVD Shell container information
 CONTAINER_NAME = avdteam/base
-DOCKER_TAG ?= 3.6
+DOCKER_TAG ?= 3.8
```
Comment (titom73) [inline development/Makefile]: Same here, better to use `3.8-v2.0`

File: .devcontainer/Dockerfile [hunk 5]
```diff
@@ -1,4 +1,4 @@
-ARG VARIANT="3.6"
+ARG VARIANT="3.8"
```
Comment (titom73) [inline .devcontainer/Dockerfile]: Same here, better to use `3.8-v2.0`

**Review by titom73 — APPROVED** [review]

File: .github/workflows/continuous_integration.yml (hunk 1 above)
Comment (ClausHolbechArista) [reply .github/workflows/continuous_integration.yml]: Fixed

File: ansible_collections/arista/avd/molecule/dhcp_provisionning/molecule.yml (hunk 2 above)
Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/molecule/dhcp_provisionning/molecule.yml]: Fixed

File: development/docker-compose.yml (hunk 3 above)
Comment (ClausHolbechArista) [reply development/docker-compose.yml]: Fixed

File: development/Makefile (hunk 4 above)
Comment (ClausHolbechArista) [reply development/Makefile]: Fixed

File: .devcontainer/Dockerfile (hunk 5 above)
Comment (ClausHolbechArista) [reply .devcontainer/Dockerfile]: Fixed

**Review by titom73 — APPROVED** [review]
LGTM!

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Pull request merged**